src/BizHawk.Tests/Client.Common/config/SerializationStabilityTests.cs

```diff
@@ -76,8 +77,13 @@ static void CheckAll<T>(string? groupDesc = null)
 				foreach (var mi in t.GetMembers(BindingFlags.Instance | BindingFlags.Public | BindingFlags.NonPublic))
 				{
 					if (mi.GetCustomAttribute<JsonIgnoreAttribute>() is not null) continue;
+					if (mi.MemberType == MemberTypes.Event) continue;
```

Comment:
I believe you don't need to touch this test after the refactor, but consider
```suggestion
					if (mi.MemberType is MemberTypes.Event) Assert.Fail("cannot serialise event {t.FullName}.{mi.Name} in {groupDesc}");
```

Reply:
I left this commit out in the new force push, since you're right it isn't needed so not relevant to this PR.
I don't see the point of explicitly failing when events are present, though. Yeah they can't be serialized but that's because there is nothing to be serialized, just like methods.